clang/lib/Headers/emmintrin.h

```diff
@@ -4745,6 +4745,77 @@ static __inline__ __m128d __DEFAULT_FN_ATTRS _mm_castsi128_pd(__m128i __a) {
   return (__m128d)__a;
 }
 
+/// Compares each of the corresponding double-precision values of two
+///    128-bit vectors of [2 x double], using the operation specified by the
+///    immediate integer operand.
+///
+///    Returns a [2 x double] vector consisting of two doubles corresponding to
+///    the two comparison results: zero if the comparison is false, and all 1's
+///    if the comparison is true.
```

Comment:
```suggestion
///    Each comparison yields 0x0 for false, 0xFFFFFFFFFFFFFFFF for true.
```
to be consistent with #83316 changes.

Reply:
71641346ebadf8d21a109813fcf66f78e0932955